soccer/planning/tests/RRTTest.cpp

```diff
@@ -0,0 +1,306 @@
+#include <gtest/gtest.h>
+
+#include "planning/low_level/RRTUtil.hpp"
+
+#if 0
```

Comment:
Why is this disabled?

Reply:
Mostly because I hadn't gotten around to re-enabling it yet. I'm not sure what the easiest way to do that with this sketchy merge strategy is, though - I may just add a pass re-enabling tests after parts 1-5 get merged into the base.